index.js

```diff
@@ -100,7 +116,7 @@ const displayPost = (post) => {
     <td class="post-posters">
       <div class="postersAvatars">${postersAvatars}</div>
     </td>
-    <td class="post-replies"></td>
+    <td class="post-replies">${getRes}</td>
```

Comment:
We have this function called `displayPost ` which has one parameter called `post`. 
The first step in understanding the problem is to understand what `post` represents. 

First add a console.log for post to see what it returns.
```js
const displayPost = (post) => {
  console.log(post)
```
This is going to log out all 30 posts with their respective data. 
Here is the first post
<img width="772" alt="Screen Shot 2022-11-19 at 10 57 41 AM" src="https://user-images.githubusercontent.com/67210629/202867062-2ff560f8-3b1f-4774-85de-ca66c61ef095.png">

Inside this `td` we can do `post.posts_count` and that will give us the following result.
<img width="1478" alt="Screen Shot 2022-11-19 at 10 59 51 AM" src="https://user-images.githubusercontent.com/67210629/202867145-977f7529-c991-4078-b98e-62a84f1b0292.png">

The reason why we don't need to setup our own map like you trying to do earlier, is because that structure has already been setup for you.  

But there is one last piece to address.

If you compare our results with the actual forum you will notice that ours is off by one. 
It looks like the API is counting all posts for that topic including the original post by the posting author.
In our case, we are only concerned with replies. 
If no one has replied it should show 0 instead of 1. 

If we modify it to say `post.posts_count - 1` then we get the correct result
<img width="1332" alt="Screen Shot 2022-11-19 at 11 05 25 AM" src="https://user-images.githubusercontent.com/67210629/202867351-52fb8d16-89ea-40fb-9bdd-79f8b305c712.png">

Reply:
Overthinking will really be the death of me. Thank you for the help!